Revision edits

jaybee84 opened the pull request:
### Purpose

Resolves #231 , #232 , #232 

### Directions for reviewers

HI All, I have ported all recent edits to GH from the word doc. Please review all changes and check if they address reviewer comments. No need to edit the word doc anymore. I was able to build the docx version locally and will be happy to do the track changes once we have finalized all edits.

#### Which areas should receive a particularly close look?

* All new edits
* Please make sure to check the Author contribution section to make sure everything looks good. 

#### Is there anything that you want to discuss further?


#### Is the pull request ready for review?

YES

### Manuscript checklist

<!--Remove this section if the changes are not relevant to the manuscript itself-->

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

- [ ] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#manubot-markdown)]
- [ ] All citations follow the [Manubot citation instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#citations)
- [ ] All changes or additions to tables follow the [Manubot table instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#tables)
- [ ] All figures follow the [Manubot figure instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#figures)
- [ ] All terms that need to be added to glossary have been added to #50 
- [ ] There are no new spelling errors identified by the Manubot spellchecker

jaybee84 committed:
porting changes from JB's edits

jaybee84 committed:
porting JB's minor edits before submission

jaybee84 committed:
adding more edits

jaybee84 committed:
port edits from RJA

jaybee84 committed:
Merge pull request #240 from jaybee84/allaway/edits021623

port edits from RJA

jaybee84 commented:
@jaclyn-taroni please do your review/edits on this PR

jaclyn-taroni committed:
Make Box 3 adhere to one sentence per line

jaclyn-taroni commented on content/16. box_3_MLtasks.md:
```diff
@@ -0,0 +1,16 @@
+### (d) Molecular subtyping of disease: 
+Rare diseases can be complicated and often show overlapping and heterogenous phenotypes. ML approaches can be used to identify molecular subtypes of the disease for better understanding. For example, unsupervised ML approaches can help identify new subtypes of a rare disease using molecular and genetic data. The same approaches can help identify the important molecular features that define the su ...
+
+### (e) Patient prognosis prediction: 
```
I don't believe this is currently referenced anywhere (source: searching all documents for "Box 3").

jaybee84 replied:
This is true. We have not touched upon Box 3e in our cited studies.

jaclyn-taroni committed:
Box 2 minor formatting edits

jaclyn-taroni committed:
Reference panel "(c)" -> "c)"

jaclyn-taroni committed:
Use "held-out validation set" everywhere

jaclyn-taroni committed:
Save a few words

jaclyn-taroni committed:
Remove use of "complex" from synopsis

jaclyn-taroni committed:
replace "cumulatively used" with "combined and used"

Per reviewer comment

jaclyn-taroni committed:
Revise wording around evaluation dataset in CV

Per reviewer comment

jaclyn-taroni committed:
Remove reference to shapes in Fig 1B legend

Per reviewer comment

jaclyn-taroni committed:
Add TODO for the LASSO reference

jaclyn-taroni committed:
Minor changes to Box 3 wording

jaclyn-taroni reviewed (commented):
Returning a comment I had here. I also wanted to ask – @jaybee84 would you be able to summarize how you picked what went into Box 3? I know we discussed using what is in the manuscript to guide what tasks to include. What I'm trying to understand: are the tasks that got picked and where they are referenced in the text are related to _citations of specific studies_? I.e., if you're referencing diagnostic decision support in the main text – "(Box 3a"), is that because one of the studies cited in that passage fit that bill? Asking to focus my review/edits 😸

jaybee84 commented:
> I.e., if you're referencing diagnostic decision support in the main text – "(Box 3a"), is that because one of the studies cited in that passage fit that bill?

@jaclyn-taroni This is exactly the case.

jaybee84 commented:
@jaclyn-taroni I realize now that I responded to only half your comment above:

The main way I arrived at the content of the box was by reviewing the main citations to see what were the major tasks those papers were referring to. Box 3a-d are cited in the sections that had references to papers which intended to do those tasks. Box 3e is the only one which is not referenced in our sections, but I added this due to my prior knowledge about a recent paper in neurofibromatosis which is partially focused on this problem.

jaclyn-taroni commented:
👍🏻  keeping this in mind as I review where the references to Box 3 are inserted and how the tasks are defined (in progress)

jaybee84 commented:
@jaclyn-taroni Thanks! If you get the chance to cut some words, that would be great too. We are currently over the word limit. I am also planning to do word count edit once your PR is in.

jaybee84 commented:
@jaclyn-taroni @allaway our one week extension request was accepted.

@jaclyn-taroni I am hoping to send the updated manuscript to Casey and Justin by Feb 22 (wed) so that they have one week to review before our submission deadline is up. Please let me know if this timeline sounds ok, or if I can help in any way to keep us on track with the timeline :)

jaclyn-taroni commented:
@jaybee84 Do you want to update the LASSO reference I added as a TODO in my branch here? That also extends to anything else that was edited in the Word doc but didn't get added here that I missed.

I am aiming to wrap up my edits today and potentially address #234 (but I think we can all work on #234 in parallel).

jaclyn-taroni committed:
Box 3 style consistent with Box 2

jaclyn-taroni committed:
Change order of boxes – move Box 3 ("tasks") up to Box 1 slot

jaclyn-taroni committed:
Add reference to common tasks in the introduction

jaclyn-taroni committed:
Make diagnostic decision support more general

jaclyn-taroni committed:
Move representation learning citations and Box 1a ref down to RL section

jaclyn-taroni committed:
Merge pull request #243 from jaybee84/jaclyn-taroni/2023-02-21-change-box-order

Change box order, make diagnostic decision support more general

jaclyn-taroni committed:
Make a few tweaks to where Box 1 is referenced

jaybee84 commented:
@jaclyn-taroni 
> Do you want to update the LASSO reference I added as a TODO in my branch here?  

-added

>That also extends to anything else that was edited in the Word doc but didn't get added here that I missed.

WIP

jaclyn-taroni committed:
Update content/05.model-complexity.md

Co-authored-by: Jineta Banerjee <jinetab@gmail.com>

jaybee84 committed:
Update 01.synopsis.md

jaybee84 committed:
Update 05.model-complexity.md

jaybee84 committed:
Update 08.outlook.md

jaybee84 commented:
> That also extends to anything else that was edited in the Word doc but didn't get added here that I missed.

@jaclyn-taroni  between your edits on #242 and my latest updates I think we have everything covered.

jaclyn-taroni committed:
Apply suggestions from code review

Co-authored-by: Jineta Banerjee <jinetab@gmail.com>

jaclyn-taroni committed:
Merge branch 'jaybee84/edits021523' into jaclyn-taroni/2023-02-19-edits

jaybee84 committed:
Merge pull request #242 from jaybee84/jaclyn-taroni/2023-02-19-edits

Jaclyn taroni/2023 02 19 edits
